shared/common-adapters/input.native.js

```diff
@@ -72,58 +58,34 @@ class Input extends Component<Props, State> {
     }
   }
 
-  _setPasswordVisible(passwordVisible: boolean) {
```

Comment:
no need to do this anymore, we pass down `secureTextEntry`

Reply:
this is unrelated to passwordVisible right: https://facebook.github.io/react-native/docs/textinput.html#keyboardtype

Reply:
? That's a `keyboardType` prop, and here we're setting a `passwordVisible` prop...

Reply:
thats cause we plumbed this through ourselves: https://github.com/keybase/client/blob/master/shared/react-native/android/app/src/main/java/io/keybase/ossifrage/components/VisiblePassReactEditTextManager.java

Reply:
BTW, this never worked -- note that the right component is `this._input`!

Reply:
Implemented a fix.